### PR TITLE
#7586: Create unstable branch of WH single card nightly FD

### PR DESCRIPTION
**Pull request by tt-rkim**

### CODE REVIEW

**Review by arakhmati — APPROVED** [review]

**Comment by AleksKnezevic** [comment]
This is a regression, I'm ok to skip test for now, but would rather not move it to unstable pipeline.

**Comment by AleksKnezevic** [comment]
Tracking here https://github.com/tenstorrent/tt-metal/issues/9158

**Comment by tt-rkim** [comment]
@AleksKnezevic just to confirm, you'd like to skip this altogether rather than have it pass ND in the unstable branch?>

**Comment by AleksKnezevic** [comment]
I would have been ok to skip until I figured out the regression, but someone beat me to it, it's fixed in latest main: 
https://github.com/tenstorrent/tt-metal/actions/runs/9386939304/job/25848917507 so I don't want to skip this test.

**Comment by tt-rkim** [comment]
This is not for ttnn unit tests - this is integration tests for SD
What was the fix? Can't find it...

File: tests/ttnn/integration_tests/stable_diffusion/test_unet_2d_condition_model.py [hunk 1]
```diff
@@ -138,6 +138,7 @@ def test_unet_2d_condition_model_256x256(device, batch_size, in_channels, input_
 
 
 @skip_for_grayskull()
+@pytest.mark.skip("#7560: Still hanging a lot")
```
Comment (AleksKnezevic) [inline tests/ttnn/integration_tests/stable_diffusion/test_unet_2d_condition_model.py:141]: Working on latest main, we shouldn't skip.

**Comment by tt-rkim** [comment]
Can't tell from the link you posted

**Comment by tt-rkim** [comment]
Here are some hangs on nightly fast dispatch on main from overnight:

https://github.com/tenstorrent/tt-metal/actions/runs/9382086403/job/25832975051 (the skipped one in this PR)
https://github.com/tenstorrent/tt-metal/actions/runs/9383668111/job/25838017615 (`test_sharded_matmuls`)

**Comment by AleksKnezevic** [comment]
The latest one seems to have passed, no: https://github.com/tenstorrent/tt-metal/actions/runs/9387431678 ?

**Comment by tt-rkim** [comment]
Yeah, so it's ND again.

"Unstable" implies non-deterministic hangs. I can choose a better name, if that's the issue

**Comment by AleksKnezevic** [comment]
ok, fair, you can move it to unstable pipeline then, but I would rather that test not get skipped.

**Comment by tt-rkim** [comment]
Sounds good - not skipping, reverting

**Review by AleksKnezevic — APPROVED** [review]

**Pull request merged**